### PR TITLE
Add a converter specified shortcut

### PR DESCRIPTION
**Pull request by noborus**
Add a shortcut specified by RAW and Align converter.

### CODE REVIEW

**Comment by ccoVeille** [comment]
Is concverter a real thing or a typo ?

File: oviewer/oviewer.go [hunk 1]
```diff
@@ -373,6 +373,12 @@ const (
 
 var Shrink rune = '…'
 
+// RawF is specifies converter shortcut for raw.
+var RawF bool
+
+// AlignF is specifies converter shortcut for align.
+var AlignF bool
```
Comment (ccoVeille) [inline oviewer/oviewer.go:380]: I don't think you need tgem to be exported

File: oviewer/oviewer.go [hunk 2]
```diff
@@ -700,6 +706,13 @@ func (root *Root) prepareRun(ctx context.Context) error {
 		root.Screen.EnableMouse(MouseFlags)
 	}
 
+	if RawF {
+		root.General.Converter = convRaw
+	}
+	if AlignF {
+		root.General.Converter = convAlign
+	}
```
Comment (ccoVeille) [inline oviewer/oviewer.go:714]: Maybe using

https://pkg.go.dev/github.com/spf13/pflag#Set

So set the converter flag?

**Review by ccoVeille — COMMENTED** [review]
Cobra is powerful framework, I'm surprised you have to do this

**Comment by noborus** [comment]
> Is concverter a real thing or a typo ?

Typo. I will fix it.

File: oviewer/oviewer.go (hunk 2 above)
Comment (noborus) [reply oviewer/oviewer.go]: Thank you for your comment.
I moved to the main side and fixed it.

**Pull request merged**